xml/x509_subjects.xml

```diff
@@ -168,6 +168,14 @@
     <param pos="0" name="os.product" value="iDRAC Linux"/>
   </fingerprint>
 
+  <fingerprint pattern="^CN=opensnort\.example.com,O=Sourcefire,emailaddress=support@sourcefire.com">
+    <description>Sourcefire Snort HTTPS console</description>
+    <example>CN=opensnort.example.com,O=Sourcefire,emailaddress=support@sourcefire.com</example>
```

Comment:
@tsellers-r7 thoughts on the ordering of email address here? Would we expect it before the CN component?

Also should it be `emailAddress`?

Reply:
Hurm, I'm unsure. Based on the intro block in this file I'd *guess* that it'd be last but it's not clear.